### PR TITLE
Omg 230 tidy and tune config

### PR DESCRIPTION
**Pull request by pdobacz**
summary of changes in the commit log messages

### CODE REVIEW

File: apps/omg_api/config/config.exs [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 config :omg_api,
-  eth_deposit_finality_margin: 10,
-  eth_submission_finality_margin: 20,
-  ethereum_event_check_height_interval_ms: 1_000,
-  rootchain_height_sync_interval_ms: 1_000,
-  child_block_minimal_enquque_gap: 4,
+  deposit_finality_margin: 10,
+  submission_finality_margin: 20,
+  ethereum_status_check_interval_ms: 6_000,
+  child_block_minimal_enquque_gap: 1,
```
Comment (pgebal) [inline apps/omg_api/config/config.exs]: enquque -> enqueue, also in mix.exs

File: apps/omg_eth/config/dev.exs [hunk 2]
```diff
@@ -1 +1,5 @@
 use Mix.Config
+
+# bumping because otherwise import/unlock of accounts will fail in some of use cases (perf tests)
+config :ethereumex,
+  request_timeout: 60_000
```
Comment (pgebal) [inline apps/omg_eth/config/dev.exs]: Maybe it's worth looking to ethereumex 0.5.1? Looks like they added those timeouts (from their github):
```
config :ethereumex,
  http_options: [timeout: 8000, recv_timeout: 5000]
```

File: docs/demo_01.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ Base.encode16(block_hash)
 
 ```bash
 # with the block hash we can get the whole block
-curl -X POST -H "Content-Type: application/json" -d '{"hash":"<block hash in hex>"}' "localhost:9656/block.get"
+echo '{"hash":"<block hash in hex>"}' | http POST "localhost:9656/block.get"
```
Comment (pgebal) [inline docs/demo_01.md]: Let's put a note about installing httpie

File: apps/omg_api/lib/application.ex [hunk 4]
```diff
@@ -36,12 +36,12 @@ defmodule OMG.API.Application do
...
            [
              %{
-               block_finality_margin: eth_deposit_finality_margin,
+               block_finality_margin: deposit_finality_margin,
                synced_height_update_key: :last_depositor_eth_height,
                service_name: :depositor,
                get_events_callback: &OMG.Eth.RootChain.get_deposits/2,
                process_events_callback: &OMG.API.State.deposit/1,
-               get_last_synced_height_callback: &OMG.Eth.RootChain.get_root_deployment_height/0
+               get_last_synced_height_callback: &OMG.DB.last_depositor_eth_height/0
```
[the remark this answers is not in the document]
Comment (pdobacz) [reply apps/omg_api/lib/application.ex]: hm, I think lower bound is guaranteed by https://github.com/omisego/elixir-omg/blob/master/apps/omg_api/lib/ethereum_event_listener.ex#L64

File: apps/omg_eth/config/dev.exs (hunk 2 above)
Comment (pdobacz) [reply apps/omg_eth/config/dev.exs]: ok, let's do grooming of ethereumex dep in a different task/chore

**Pull request merged**